scss/jquery/overrides/_select2.scss

```diff
@@ -0,0 +1,246 @@
+// Single select2
+.select2-container {
+  // The input field overlay
+  .select2-choice {
+    background: $crm-white;
+    border-color: $crm-grayblue-darker;
+    border-radius: $border-radius-base;
+    box-shadow: $crm-form-control-shadow;
+    box-sizing: border-box;
+    height: $input-height-base;
+    line-height: 27px;
```

Comment:
Why `27px`?

Reply:
The line-height needs to be lower than the actual height of the element, otherwise the text inside the select2 will not be properly centered vertically

<img width="350" alt="line-height" src="https://user-images.githubusercontent.com/6400898/46876190-f98e9c80-ce3d-11e8-9112-d1dff0283caf.png">

_(with `line-height: $input-height-base;`)_

Reply:
I can do
```scss
line-height: $input-height-base - 3;
```
what do you think

Reply:
created a variable